### DISCUSSION TITLE
Multiple tabs on dashboard view

### DISCUSSION THREAD

**Discussion by kovacbb** [spatie/laravel-dashboard-skeleton-tile#6] [Q&A] [answered]
Hello,
As I'm planing to make 30+ tiles, with more then just simple output (charts, lists, etc), how can I extend dashboard view and sort them per tabs?
Also is it possible to enable scrooling on dashboard as there is too much data for a singe page view?

Thank you!

**Comment by freekmurze** [comment, marked as answer]
My short answer would be is that currently this isn't possible, the grid always tries to go full screen.
@willemvb is there a simple solution using CSS?

**Reply by willemvb** [reply]
We don't have a tabs solution out-of-the-box, but scrolling should be doable.

You can publish the views as documented here: 
https://spatie.be/docs/laravel-dashboard/v2/basic-usage/customizing-the-views

From there you can edit the main layout on this line https://github.com/spatie/laravel-dashboard/blob/master/resources/views/dashboard.blade.php#L22
to remove the `h-screen` and `fixed` class. Then set a new height on this element (eg. `height:300vh`) to trigger scrolling.

**Reply by kovacbb** [reply]
thank you
